lib/evss/jwt.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+require 'jwt'
+require 'securerandom'
+
+module EVSS
+  class Jwt
+    ISSUER            = 'Vets.gov'
+    SIGNING_ALGORITHM = 'HS256' # HMAC SHA256
+    SIGNING_KEY       = 'my$ecretK3y' # TODO: integrate into Settings + devops
+    EXP_WINDOW        = 5.minutes.to_i
+
+    def initialize(user)
+      @user = user
+    end
+
+    def encode
+      JWT.encode(payload, SIGNING_KEY, SIGNING_ALGORITHM, typ: 'JWT')
+    end
+
+    private
+
+    def payload
+      {
+        # reserved claims
+        iat: Time.now.to_i,
+        exp: Time.now.to_i + EXP_WINDOW,
```

Comment:
might want `exp` to be a setting, or check with EVSS what their preferred window is, but five minutes is a good baseline.

Reply:
Confirmed that 5 minutes is correct. It's actually defined in their doc, I must have missed it.

Going to leave this hard-coded for now.  It can be made into a setting quite easily but I don't foresee us changing it,